README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# __*Homework 1*__
+* __Name__: Moksha Sharma
+* __Date of Birth__: 12 November 1998  
+* __Place of Birth__: New Delhi, India  
+* __Education__: 
+    + __High School Diploma__: Delhi Public School Rohini
+    + __Bachelor of Arts__: University of Chicago   
+
+I am a rising third year student at the University of Chicago and am pursuing a double major in political science and English. I am particularly interested in rhetoric and political philosophy. In addition to my majors, I also  enjoy studying languages and film theory, and I intend to take taking more classes in both during the remainder of my time at the University. The attached is the link to a photograph that was analysed in a film class that I took last quarter. The discussion about the photograph taught me a lot about the specific qualities of film as a medium and made me appreciate visual art in a way I had never considered before. 
+
+<img>[Art Institute II](https://images.app.goo.gl/oTxHndcSeNdjrPnD6)<img>
```

Comment:
Did you want to embed the image in your README? If so, try this code:

```
![Art Institute II](https://www.artic.edu/iiif/2/ff4d0227-83df-35f6-ebd9-fbcb483af2df/full/1200,/0/default.jpg?w=1200&h=800&fit=crop)
```

![Art Institute II](https://www.artic.edu/iiif/2/ff4d0227-83df-35f6-ebd9-fbcb483af2df/full/1200,/0/default.jpg?w=1200&h=800&fit=crop)

You just needed to get the correct URL for the image file directly, not the Google search results page